prow/config/config.go

```diff
@@ -898,16 +928,26 @@ func (c *Config) finalizeJobConfig() error {
 			return errors.New("no default GCS credentials secret provided for plank")
 		}
 
+		if c.Plank.DefaultDecorationConfig != nil && len(c.Plank.DefaultDecorationConfigs) == 0 {
+			logrus.Warning("default_decoration_config will be deprecated in the future and it will be replaced with default_decoration_configs['*'].")
```

Comment:
Can we give an exact date for the deprecation? These vague promises tend to rot...

Reply:
Yes, that kind of information should be required. @stevekuznetsov any thoughts on the date? Do we usually give 6 months notice?

Reply:
Yep, 6mo